fluent-package/msi/assets/fluent-package-post-migration.bat

```diff
@@ -0,0 +1,51 @@
+@echo on
+title Fluent-package post migration script
+
+if exist "%~dp0..\..\td-agent" (
+  setlocal enabledelayedexpansion
+  sc query state=inactive | findstr fluentdwinsvc
+  if !ERRORLEVEL! == 1 (
+    @rem NOTE: If the service is not stopped here, it is a BUG.
+    @rem This state can cause fatal problems with the installation, so we should cancel this process.
+    @rem (If trying to stop it by the command such as `$ net stop fluentdwinsvc`, it can fail as "The service is starting or stopping.  Please try again later.".)
+    echo "WARN: Active fluentdwinsvc service is detected. This is unexpected. Migration process has been canceled."
+    exit /b 1

```

Comment:
I checked this behavior by rewriting the codes.
When exiting ERRORLEVEL `1`, Rollback doesn't occur, and the installation is completed without running the migration process in this script.

I think this is acceptable for now.
Fixes to ensure Rollback should be done in future work.

Reply:
I think so too.

Reply:
Thanks for the confirmation. :+1: